src/tests/auth.test.ts

```diff
@@ -11,7 +11,14 @@ const app = new App([new AuthRoute()]);
 const authRoute = new AuthRoute();
 const authService = new AuthService();
 
-const testCookie = authService.createTestCookie();
+const testAccessCookie = authService.createTestCookie('access', 'access');
+const testRefreshCookie = authService.createTestCookie('refresh', 'refresh');
+const testAccessCookieShortSpan = authService.createTestCookie('shortSpanTestAccess', 'access');
+const testRefreshCookieShortSpan = authService.createTestCookie('shortSpanTestRefresh', 'access');
+
+//Time out is set to ensure that short span cookies will expire
+setTimeout(() => {}, 1500);
```

Comment:
I'm pretty sure this is running before beforeeach. Maybe do it in AfterEach or AfterAll